lib/cs_guide/resources/venue.ex

```diff
@@ -12,8 +12,16 @@ defmodule CsGuide.Resources.Venue do
     field(:postcode, :string)
     field(:phone_number, :string)
     field(:cs_score, :float, default: 0.0)
+    field(:num_cocktails, :integer)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
+    field(:description, :string)
```

Comment:
@Danwhy all of these fields should be using the `Fields.field_name` to ensure that we are doing proper validation/sanitisation. please see: https://github.com/dwyl/fields/issues/20
thanks! ✨